QLogger.h

```diff
@@ -43,10 +43,13 @@ class QLoggerManager
     * @brief Gets an instance to the QLoggerManager.
     * @return A pointer to the instance.
     */
-   static QLoggerManager *getInstance();
+   static QLoggerManager &getInstance();
```

Comment:
Documentation needs to be fixed. This is no longer a pointer.

Reply:
I can do that ofcourse, but I thought since this is your project you would like to do this yourself?

Reply:
It's fine you do it as the method is being changed in the PR.